src/fluree/server/handler.clj

```diff
@@ -62,6 +62,12 @@
 (def TransactRequestBody
   (m/schema [:map-of :any :any]))
 
+(def ImportRequestBody
+  (m/schema [:or
+             [:string {:min 1}]
+             [:map-of :any :any]
+             [:sequential map?]]))
```

Comment:
Just checking to make sure I understand, these schemas correspond to:
- turtle string
- json-ld map
- vector of json-ld maps

Is that right?